README.md

```diff
@@ -1,19 +1,31 @@
 # Pipeline Service
 
-Pipeline Service provides a SaaS for pipelines. It leverages Kubernetes, Tekton, Tekton Triggers, Tekton Results, Pipelines as Code together with kcp.
+Pipeline Service provides a SaaS for pipelines. It leverages:
+
+- kcp for serving the API and acting as control plane
+- Kubernetes / OpenShift for the compute
+- Tekton Pipelines, Results and friends for the core of the service
+- OpenShift GitOps / Argo CD, Pipelines as Code for managing the infrastructure
+
 
 ## Why Pipeline Service?
 
 Tekton and Kubernetes provide a great infrastructure for build pipelines. They come however with some limitations.
 
-- Multi-tenancy: Kubernetes provides a level of multi-tenancy. CustomResourceDefinitions are however cluster scoped. This induces a coupling between the operator version provided by the platform and what you can use as a tenant. The control plane is also shared between tenants.
+- Multi-tenancy: Kubernetes provides a level of multi-tenancy. This does however not extend to cluster scoped resources. CustomResourceDefinitions (CRD) are extensively used for extending the Kubernetes API, following [the operator pattnern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/).  CRDs are cluster scoped. This induces a coupling between the operator version provided by the platform and what you can use as a tenant. The control plane is also shared between tenants.
```

Comment:
`This does however not extend to` -> `However this does not extend to`
`pattnern` -> `pattern`